### PR TITLE
Upload logs to google drive on Travis-CI failures

### PR DESCRIPTION
**Pull request by DaoWen**
## Changes proposed in this PR

Upload logs to Google Drive when an integration test fails.

## Why are we making these changes?

The logs are too big to dump to stdout (they get truncated, and it takes *forever*).

### CODE REVIEW

File: waiter/bin/ci/upload_logs.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    docker ps --all --last 10
+
+    # Extract the Mesos master logs from docker
+    mesos_master_container=$(docker ps --all --latest --filter 'name=minimesos-master-' --format '{{.ID}}')
+    mkdir -p ./mesos/master-logs
+    docker cp --follow-link $mesos_master_container:/var/log/mesos-master.INFO ./mesos/master-logs/
+    docker cp --follow-link $mesos_master_container:/var/log/mesos-master.WARNING ./mesos/master-logs/
+
+    # Extract the Mesos agent logs from docker
+    mesos_agent_containers=$(docker ps --all --last 6 --filter 'name=minimesos-agent-' --format '{{.ID}}')
```
Comment (dposada) [inline waiter/bin/ci/upload_logs.sh]: Are we running with 6 minimesos agents?

Comment (DaoWen) [reply waiter/bin/ci/upload_logs.sh]: It looks like we only have 4. Fixed.

**Comment by dposada** [comment]
Is this still WIP?

**Comment by DaoWen** [comment]
@dposada - The failed test log uploads were successful. You should see two of them in the new Waiter-CI Travis-Dumps drive folder I shared with you earlier.

Unless you have additional feedback, then this is ready to merge.

**Comment by DaoWen** [comment]
Closes #382.

**Pull request merged**